src/app/session.cpp

```diff
@@ -494,8 +502,8 @@ void Session::update()
         qApp->setStyleSheet(this->core_.stylesheet(this->theme_.mode, this->layout_.scale));
 
         for (QWidget *widget : qApp->allWidgets()) {
-            auto button = qobject_cast<QAbstractButton*>(widget);
-            if ((button != nullptr) && !button->property("iconized").isNull())
+	  if( auto button = qobject_cast<QAbstractButton*>(widget) )
+            if (button->property("colorized").toBool())
```

Comment:
This change is a bugfix. We were errantly creating masks and setting icons on a bunch of controls. This speeds up dark/light mode switching by quite a bit.

Reply:
well that might be what is causing the 1 sec+ delay when switching colors or toggling dark mode :p